script/prebuild.js

```diff
@@ -2,12 +2,13 @@ const semver = require('semver');
 const fs = require('fs');
 const path = require('path');
 
-const minimumNodeVersion = '8.10.0';
+const nodeVersion = path.join(__dirname, '../.nvmrc');
+const minimumNodeVersion = fs.readFileSync(nodeVersion).toString();
```

Comment:
I made a small update to read from the `.nvmrc` instead of just hardcoding the Node version.